src/lib/govuk/Checkbox.svelte

```diff
@@ -1,19 +1,17 @@
 <script lang="ts">
   // This component must be placed inside a CheckboxGroup
 
-  export let label: string;
```

Comment:
I changed the API of this component, because we want to show the little colored box next to the label. There was only one other user, in the route controls, and it's not harder to use